meta/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+galaxy_info:
+  author: "David Reed"
```

Comment:
Do we need it?

Reply:
I think it's required for Galaxy, could change to a generic value of 'Sansible' maybe?